Build is broken

Building on humble. 0.4.0 works, but from head as of today, I get the following error: `error: ‘tbb::info’ has not been declared`

```
11490.2 Starting >>> kiss_icp
11509.4 --- stderr: kiss_icp
11509.4 CMake Deprecation Warning at /opt/ros/humble/build/kiss_icp/_deps/sophus-src/CMakeLists.txt:1 (cmake_minimum_required):
11509.4   Compatibility with CMake < 3.5 will be removed from a future version of
11509.4   CMake.
11509.4 
11509.4   Update the VERSION argument <min> value or use a ...<max> suffix to tell
11509.4   CMake that the project does not need compatibility with older versions.
11509.4 
11509.4 
11509.4 CMake Deprecation Warning at /opt/ros/humble/build/kiss_icp/_deps/tessil-src/CMakeLists.txt:1 (cmake_minimum_required):
11509.4   Compatibility with CMake < 3.5 will be removed from a future version of
11509.4   CMake.
11509.4 
11509.4   Update the VERSION argument <min> value or use a ...<max> suffix to tell
11509.4   CMake that the project does not need compatibility with older versions.
11509.4 
11509.4 
11509.4 CMake Warning:
11509.4   Manually-specified variables were not used by the project:
11509.4 
11509.4     HUNTER_KEEP_PACKAGE_SOURCES
11509.4 
11509.4 
11509.4 /opt/ros/humble/src/kiss-icp/cpp/kiss_icp/core/Registration.cpp: In constructor ‘kiss_icp::Registration::Registration(int, double, int)’:
11509.4 /opt/ros/humble/src/kiss-icp/cpp/kiss_icp/core/Registration.cpp:174:69: error: ‘tbb::info’ has not been declared
11509.4   174 |       max_num_threads_(max_num_threads > 0 ? max_num_threads : tbb::info::default_concurrency()) {
11509.4       |                                                                     ^~~~
11509.4 make[2]: *** [kiss_icp/core/CMakeFiles/kiss_icp_core.dir/build.make:76: kiss_icp/core/CMakeFiles/kiss_icp_core.dir/Registration.cpp.o] Error 1
11509.4 make[1]: *** [CMakeFiles/Makefile2:322: kiss_icp/core/CMakeFiles/kiss_icp_core.dir/all] Error 2
11509.4 make: *** [Makefile:146: all] Error 2
11509.4 ---

```

--- a/ros/CMakeLists.txt
+++ b/ros/CMakeLists.txt
@@ -23,9 +23,7 @@
 cmake_minimum_required(VERSION 3.16...3.26)
 project(kiss_icp VERSION 0.4.0 LANGUAGES CXX)
 
-set(ignore ${CATKIN_INSTALL_INTO_PREFIX_ROOT})
 set(CMAKE_BUILD_TYPE Release)
-set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
 if(EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/../cpp/kiss_icp/)
   add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/../cpp/kiss_icp ${CMAKE_CURRENT_BINARY_DIR}/kiss_icp)
@@ -41,11 +39,13 @@ else()
 endif()
 
 find_package(ament_cmake REQUIRED)
+find_package(geometry_msgs REQUIRED)
 find_package(nav_msgs REQUIRED)
-find_package(rcutils REQUIRED)
 find_package(rclcpp REQUIRED)
 find_package(rclcpp_components REQUIRED)
+find_package(rcutils REQUIRED)
 find_package(sensor_msgs REQUIRED)
+find_package(std_msgs REQUIRED)
 find_package(tf2_ros REQUIRED)
 
 # ROS 2 node
@@ -55,12 +55,13 @@ target_include_directories(odometry_component PRIVATE ${CMAKE_CURRENT_SOURCE_DIR
 target_link_libraries(odometry_component kiss_icp_pipeline)
 ament_target_dependencies(
   odometry_component
-  rcutils
+  geometry_msgs
+  nav_msgs
   rclcpp
   rclcpp_components
-  nav_msgs
+  rcutils
   sensor_msgs
-  geometry_msgs
+  std_msgs
   tf2_ros)
 
 rclcpp_components_register_node(odometry_component PLUGIN "kiss_icp_ros::OdometryServer" EXECUTABLE kiss_icp_node)

--- a/ros/package.xml
+++ b/ros/package.xml
@@ -32,15 +32,19 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <depend>rcutils</depend>
-  <depend>rclcpp</depend>
-  <depend>rclcpp_components</depend>
   <depend>geometry_msgs</depend>
   <depend>nav_msgs</depend>
-  <depend>std_msgs</depend>
+  <depend>rclcpp</depend>
+  <depend>rclcpp_components</depend>
+  <depend>rcutils</depend>
   <depend>sensor_msgs</depend>
-  <depend>tf2</depend>
+  <depend>std_msgs</depend>
   <depend>tf2_ros</depend>
+
+  <depend>eigen</depend>
+  <depend>sophus</depend>
+  <depend>tbb</depend>
+  <depend>robin-map-dev</depend>
 
   <exec_depend>ros2launch</exec_depend>
 

